srcs/hooks/hooks.c

```diff
@@ -58,21 +60,25 @@ int	key_down(int keycode, t_cub *cub)
 int update_display(t_cub *cub)
 {
 	static int frames = 0;
+	double frame_time;
 
-	if (frames++ == 1000)
+	cub->old_time = cub->time;
+	cub->time = get_time_micros();
+	frame_time = (double)(cub->time - cub->old_time) / 1000; // in milliseconds
+	if (frames++ == 15)
```

Comment:
15 is too fast for me, it will break

Reply:
I can start doing some research on this because I am now at school and can test between both OS. But it seems there are some significant differences I hope I can fix..

Just to clarify as well: the frames variable is used so we don't do the calculation of the rays every mlx_loop_hook. It seems to me that as raycasting is an old technique, it runs extremely fast on modern computers and is also why our FPS is in the 10,000s. If we were implementing a raytracer instead, our FPS would probably be much lower as every individual pixel needs to be animated rather than every column of pixels.

Reply:
Truth is, my computer was hyperventilating everytime I ran our cub programme, and it stoped doing that with this new implementation of yours. So for sure it is optimized. I just think we need to rethink that number over there. As I said, the best one for my computer is 100. But I would like to make it dependent on the frame_time, or from some other variable that may change with the device, yet my attempts failed.